docs/reference/logging/overview.md

```diff
@@ -41,6 +43,21 @@ Logs that reach `max-size` are rolled over. You can set the size in kilobytes(k)
 
 If `max-size` and `max-file` are set, `docker logs` only returns the log lines from the newest log file.
 
+The `labels` and `env` options adds additional attributes to be logged
```

Comment:
@vdemeester Some of this isn't clear.  I think examples with actual commands will help. I've taken a stab at it.

--->edit for markdown

The `labels` and `env` options add additional attributes for use with this logging driver. Each of these options takes a comma-separated list of
keys. If there is collision between `label` and `env` keys, the value of the
`env` takes precedence.

To use attributes, specify them when you start the Docker daemon.

```
docker daemon --log-driver=json-file --log-opt labels=foo --log-opt env=foo,fizz
```

Then, run a container and specify values for the `labels` or `env`.  For example, you might use this:

```
docker run --label foo=bar -e fizz=buzz -d -P training/webapp python app.py
````

This adds  field that looks like:

    "attrs":{"fizz":"buzz","foo":"bar"}

Use `docker inspect CONTAINERNAME` to view these values.